extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -110,6 +118,12 @@ const renderDefaultPageAttachmentUI = (pageEl, attachmentEl) => {
 const renderPageAttachmentUiWithImages = (win, pageEl, attachmentEl) => {
   const openAttachmentEl = buildOpenInlineAttachmentElement(pageEl);
 
+  // Setting theme
+  const theme = attachmentEl.getAttribute('theme');
+  if (theme && AttachmentTheme.DARK === theme.toLowerCase()) {
```

Comment:
You could enforce with validation that the value of `theme` is always lowercase `light` or `dark`, so you can just propagate the attribute without requiring the AttachmentTheme enum or any other checks. A reason to keep it like this is if you plan to add more JS logic over the theme, but if it's purely CSS it could be solved with

`openAttachmentEl.setAttribute("theme", attachmentEl.getAttribute(theme))`

Reply:
These are actually already case sensitive in the validator: https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/validator-amp-story.protoascii#L856

Bc of that, I've removed all the AttributeTheme manual change stuff here. Thanks!